src/core/model/types/mixed_type.h

```diff
@@ -22,11 +22,26 @@ namespace model {
  */
 class MixedType final : public Type {
 private:
+    template <typename T>
+    struct TupleMaxAlign {};
+    template <typename... Ts>
+    struct TupleMaxAlign<std::tuple<Ts...>> {
+        static constexpr size_t value = std::max({alignof(Ts)...});
+    };
```

Comment:
unused indeed, it was left from the previous implementation

Reply:
> And can't this be done using template variables?

It's technically possible to implement this using template variables, but what value should I use for a generic case? `struct` is used because it's possible to "narrow" a metafunction domain to some specific types, like in this case `TupleMaxAlign` has `value` member only if template parameter is a `tuple`

Reply:
I thought it could be used to get the maximum alignment, so there would be only one usage of it, with no need to think about anything else. I guess it's not important now.

Reply:
I thought it could be used to get the maximum alignment, so there would be only one usage of it, with no need to think about anything else. I guess it's not important now.